tests/llmobs/test_llmobs_service.py

```diff
@@ -1359,6 +1359,31 @@ def test_llmobs_fork_recreates_and_restarts_span_writer():
         llmobs_service.disable()
 
 
+def test_llmobs_fork_recreates_and_restarts_agentless_span_writer():
+    """Test that forking a process correctly recreates and restarts the LLMObsSpanWriter."""
+    with override_global_config(dict(_dd_api_key="<not-a-real-key>")):
+        with mock.patch("ddtrace.internal.writer.HTTPWriter._send_payload"):
+            llmobs_service.enable(_tracer=DummyTracer(), ml_app="test_app", agentless_enabled=True)
+            original_pid = llmobs_service._instance.tracer._pid
+            original_span_writer = llmobs_service._instance._llmobs_span_writer
+            pid = os.fork()
+            if pid:  # parent
+                assert llmobs_service._instance.tracer._pid == original_pid
```

Comment:
don't need to assert on the tracer pid, that's out of scope for this functionality

Reply:
oop yea good point i just copy/pasta'd 😅